ffmpeg-plugin/clone-and-patch-ffmpeg.sh

```diff
@@ -9,8 +9,8 @@ SCRIPT_DIR="$(readlink -f "$(dirname -- "${BASH_SOURCE[0]}")")"
 BUILD_DIR="${BUILD_DIR:-${SCRIPT_DIR}/build}"
 . "${SCRIPT_DIR}/../common.sh"
 
-# Default to latest 6.1
-FFMPEG_VER="${FFMPEG_VER:-6.1}"
+# Default to latest 7.0
+FFMPEG_VER="${FFMPEG_VER:-7.0}"
```

Comment:
Please change this to latest patch version as there are lots of fixes there.

```suggestion
# Default to latest 7.0.2
FFMPEG_VER="${FFMPEG_VER:-7.0.2}"
```

Reply:
I don't get it. We're cloning FFmpeg from the release branch, not the tag
```bash
# Clone FFmpeg:
git clone --branch "release/${FFMPEG_VER}" --depth 1 https://github.com/FFmpeg/FFmpeg.git "${BUILD_DIR}/FFmpeg"
```
This means if you clone it today, it will include all commits related to `7.0.2` issued recently.